opentelekomcloud/services/vpc/resource_opentelekomcloud_vpc_v1.go

```diff
@@ -193,6 +265,12 @@ func resourceVirtualPrivateCloudV1Read(ctx context.Context, d *schema.ResourceDa
 		return diag.FromErr(err)
 	}
 
+	if _, ok := d.GetOk("secondary_cidr"); ok {
```

Comment:
Redundant check, "secondary_cidr" should be always read regardless of provided config

Reply:
changed